sirepo/jupyterhub.py

```diff
@@ -34,6 +35,9 @@ async def authenticate(self, handler, data):
         _set_cookie(handler)
         try:
             sirepo.auth.require_user()
+            sirepo.auth.require_sim_type('jupyterhublogin')
+        except werkzeug.exceptions.Forbidden:
+            return None
```

Comment:
this needs explanation. what does None mean here?

Reply:
83ca09c4